clang/lib/StaticAnalyzer/Checkers/GenericTaintChecker.cpp

```diff
@@ -1046,10 +1044,7 @@ bool GenericTaintChecker::generateReportIfTainted(const Expr *E, StringRef Msg,
     return false;
 
   // Generate diagnostic.
```

Comment:
:thinking: Perhaps add an `assert(BT)` here for the sake of paranoia?

Reply:
If we are there, maybe we should also assert that inside the register call, before emplacing it was empty.